core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -1545,6 +1547,10 @@ private void updateUrlProcessor() {
     }
   }
 
+  private void updateNightMode() {
+    painter.update(getCurrentWebView(), { getCurrentWebView().getUrl() == null || getCurrentWebView().getUrl() != HOME_URL}, videoView);
```

Comment:
@macgills here I'm trying to call the `update` function.

Reply:
you don't need to call `getCurrentWebview()` in that lambda,

```
    shouldActivateCriteria: ((T) -> Boolean) = { true }
```

`(T)->Boolean` translation: this parameter is a function which takes a view as a parameter and returns a Boolean. In kotlin when a lambda only has 1 parameter you don't have to declare it explicitly and can instead reference it by `it` in the lambda scope.

To be explicit what you wrote could be rewritten as 
```
{ theWebviewYouPassedAsTheFirstParamter ->
getCurrentWebView().getUrl() == null || getCurrentWebView().getUrl() != HOME_URL
}
```

ahhh but this is java. delete the lambda, type `new` press ctrl+space and choose the thing that looks most appropriate to create in this position. See DialogShower usage in java for an example